templates/taxi/confirm_delete_manufacturer.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <p>Do you want to delete?{{ car }}</p>
+  <p><i>Everything what bundle with this manufacturer will be also deleted!</i></p>
+  <form action="" method="post">
+  {% csrf_token %}
+    <input type="submit" value="Confirm" class="btn btn-danger">
+    <a href="{% url 'taxi:manufacturer-list' %}" class="btn btn-secondary">Cancel</a>
```

Comment:
The 'Cancel' button should redirect to the previous page, not the manufacturer list page, to improve user experience. Consider using JavaScript to go back to the previous page or adjust the URL accordingly.